gradle-jdks-test-common/src/main/groovy/com/palantir/gradle/jdks/GradleJdkTestUtils.groovy

```diff
@@ -17,12 +17,11 @@
 package com.palantir.gradle.jdks
 
 import com.palantir.gradle.jdks.setup.common.CommandRunner
-import nebula.test.IntegrationSpec
 import org.apache.commons.lang3.tuple.Pair
 
 import java.nio.file.Path
 
-abstract class GradleJdkIntegrationTest extends IntegrationSpec {
+class GradleJdkTestUtils {
```

Comment:
refactored this such that it can be used by the tests in both gradle-jdks & gradle-jdks-settings.